sections/03_categorize.md

```diff
@@ -46,65 +46,167 @@ regards to data availability and labeling.
-all of this work, the researchers must work around a specific challenge - the
-limited number of well annotated training images. To expand the number and
-diversity of images, the researchers have employed approaches where they employ
-adversarial examples [@doi:10.1101/095786] or first train towards human-created
-features before subsequent fine tuning [@doi:10.1007/978-3-319-46723-8_13]. The
-presence of a large bank of well-annotated mammography images would aid in the
-application of deep neural networks to this area. Though this strategy has not
-yet been employed in this domain, large collections of unlabeled images might
-first be used in an unsupervised context to construct high-quality feature
-detectors. Then the small number of labeled examples could be used for
-subsequent training. Similar strategies have been employed for EHR data where
-high-quality labeled examples are also difficult to obtain
-[@doi:10.1101/039800].
-
-In addition to radiographic images, histology slides are also being analyzed
-with deep learning approaches. Ciresan et al.
-[@doi:10.1007/978-3-642-40763-5_51] developed one of the earliest examples,
-winning the 2012 International Conference on Pattern Recognition's Contest on
-Mitosis Detection while achieving human competitive accuracy. Their approach
-uses what has become a standard convolutional neural network architecture
-trained on public data.  In more recent work,  Wang et al.[@arxiv:1606.05718]
-analyzed stained slides to identify cancers  within slides of lymph node slices.
-The approach provided a probability map for  each slide. On this task a
-pathologist has about a 3% error rate. The  pathologist did not produce any
-false positives, but did have a number of false  negatives. Their algorithm had
-about twice the error rate of a pathologist.  However, their algorithms errors
-were not strongly correlated with the  pathologist. Theoretically, combining
-both could reduce the error rate to  under 1%. In this area, these algorithms
-may be ready to incorporate into  existing tools to aid pathologists. The
-authors' work suggests that this could  reduce the false negative rate of such
-evaluations. This theme of an ensemble  between deep learning algorithm and
-human expert may help overcome some of the  challenges presented by data
-limitations.
+imaging extensively are also emerging. To the date, deep learning has been
+employed for a wide range of tasks in medical imaging, including classification
+of exams and lesions/nodules, localization of organs, regions, landmarks, and
+lesions, segmentation of organs, organ substructures, and lesions, medical
+image registration, content-based image retrieval, image generation and
+enhancement, and combining image data with clinical reports. For the extensive
+literature review, including approaches and applications see recent papers
+[@tag:Litjens2017_medimage_survey @tag:Shen2017_medimg_review].
```

Comment:
I think just tag these references on the end of the previous sentence and save a line.